rollup.config.js

```diff
@@ -25,8 +25,9 @@ module.exports = [
 		plugins: [
 			resolve(),
 			commonjs(),
-			babel({
-				exclude: 'node_modules/**'
+			buble({
+				objectAssign: 'Object.assign',
```

Comment:
What does this do?

Reply:
We're required to pollyfill `Object.assign`. This pollyfills `Object.assign` with `Object.assign` since the only place we use `Object.assign` we already have our own fallback in place